.eslintrc.js

```diff
@@ -0,0 +1,312 @@
+        "@typescript-eslint/unified-signatures": "error",
+        "arrow-parens": [
+            "error",
+            "always"
+        ],
+        "complexity": "off",
+        "constructor-super": "error",
+        "curly": "error",
+        "default-case": "error",
+        "dot-notation": "error",
+        "eol-last": "off",
+        "guard-for-in": "error",
+        "max-classes-per-file": [
+            "error",
+            1
+        ],
+        "member-ordering": "off",
+        "new-parens": "error",
+        "no-bitwise": "error",
+        "no-caller": "error",
+        "no-cond-assign": "error",
+        "no-console": [
+            "error",
+            {
+                "allow": [
+                    "error",
+                    "debug",
+                    "info",
+                    "time",
+                    "timeEnd",
+                    "trace",
+                    "warn"
+                ]
+            }
+        ],
+        "no-constant-condition": "error",
+        "no-control-regex": "error",
+        "no-debugger": "error",
+        "no-duplicate-case": "error",
+        "no-empty": "error",
+        "no-empty-function": "off",
+        "no-eval": "error",
+        "no-extra-bind": "error",
+        "no-extra-semi": "error",
+        "no-fallthrough": "off",
+        "no-invalid-regexp": "error",
+        "no-invalid-this": "error",
+        "no-multi-str": "off",
+        "no-multiple-empty-lines": "error",
+        "no-new-func": "error",
+        "no-new-wrappers": "error",
+        "no-octal": "error",
+        "no-octal-escape": "error",
+        "no-regex-spaces": "error",
+        "no-return-await": "error",
+        "no-sequences": "error",
+        "no-sparse-arrays": "error",
+        "no-template-curly-in-string": "error",
+        "no-throw-literal": "error",
+        "no-undef-init": "error",
+        "no-unsafe-finally": "error",
+        "no-unused-labels": "error",
+        "no-var": "error",
+        "object-shorthand": "error",
+        "one-var": "off",
+        "prefer-const": "error",
+        "prefer-object-spread": "error",
+        "quote-props": [
+            "error",
+            "consistent-as-needed"
+        ],
+        "radix": "error",
+        "space-before-function-paren": ["error", "never"],
+        "use-isnan": "error",
+        "valid-typeof": "off",
+        "@typescript-eslint/tslint/config": [
+            "error",
+            {
+                "rulesDirectory": [
+                    "node_modules/tslint-react/rules",
+                    "node_modules/tslint-microsoft-contrib",
+                    "node_modules/@crazyfactory/tslint-rules/lib"
+                ],
+                "rules": {
+                    "align": [
+                        true,
+                        "parameters",
+                        "arguments",
+                        "statements"
+                    ],
+                    "chai-vague-errors": true,
```

Comment:
I think chai doesn't need to be there for us? I think it's when you pass the error message yourself, but jest doesn't allow that

Reply:
Same, it's old rule we had in tslint.json. Yes, this can be removed.